snapshots/cardano-1.25.1-GHC-8.10.4.yaml

```diff
@@ -0,0 +1,155 @@
+name: cardano-1.25.1-GHC-8.10.4
+
+resolver: lts-17.4
+
+packages:
+- base16-0.1.2.1
+- base16-bytestring-1.0.1.0
+- base58-bytestring-0.1.0
+- base64-0.4.2
+- bech32-1.1.0
```

Comment:
https://www.stackage.org/lts-17.6/package/bech32-1.1.0

Reply:
I'll leave bech32 in there because it's one of ours.